stable/console-chart/templates/console-clusterrole.yaml

```diff
@@ -41,9 +44,62 @@ rules:
   verbs:
   - list
   - get
+
 - apiGroups:
   - ''
   resources:
   - 'services'
   verbs:
   - list
+
+- verbs:
+  - watch
```

Comment:
only watch, but don't need get & list?

Reply:
The service account only watches.  The user token is used for all other access such as get and list.